default.nix

```diff
@@ -341,7 +344,7 @@ in let this = rec {
       '';
     };
   };
-  ghcjsCompiler = (overrideCabal (ghc.callPackage "${nixpkgs.path}/pkgs/development/compilers/ghcjs" {
+  ghcjsCompiler = (overrideCabal (ghc.callPackage (nixpkgs.path + "/pkgs/development/compilers/ghcjs/base.nix") {
```

Comment:
I think this requires a newer version of nixpkgs than reflex-platform uses.  I'll update it in my merge.